extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -89,8 +89,7 @@ Per-vertex feature IDs can be used to identify individual objects that have been
 >         "EXT_mesh_features": {
 >           "featureIds": [{
 >             "featureCount": 2,
->             "attribute": 0,
->             "label": "buildings"
+>             "attribute": 0
```

Comment:
I removed the text from the feature-id-by-attribute diagram above, partly for aesthetic reasons, but also because its redundant with the text above.

Before | After
--|--
![2](https://user-images.githubusercontent.com/915398/158079512-cbdeaa68-0c8a-4f5f-9b4c-5de1f66f06eb.png)|![1](https://user-images.githubusercontent.com/915398/158079511-85749e6b-464f-4e4d-9741-8dc189ffbd19.png)